apache/trafficcontrol
# Rocky Linux 8 support

This PR adds Rocky Linux 8 support and drops CentOS 8 support, since CentOS 8 is EOL as of 12-31-2021. Targeting CentOS 7 is still supported.
<hr/>

## Which Traffic Control components are affected by this PR?

- Documentation
- Traffic Monitor - integration tests
- CDN in a Box
- Automation - Docker images, GitHub Actions
- Build system

## What is the best way to verify this PR?

1. Build the builder images and build the RPMs
   ```shel0
   cd infrastructure/cdn-in-a-box
   make -j build-builders
   ```
2. Ensure CDN in a Box runs successfully
3. Check other modified docker-compose environments around the repo

## If this is a bugfix, which Traffic Control versions contained the bug?

- 5.1.5
- 5.1.6 (RC0)
- 6.0.2
- 6.1.0 (RC2)
- master (f5de844341)

## PR submission checklist
- [x] Existing tests are sufficient
